shell/pages/c/_cluster/settings/brand.vue

```diff
@@ -261,6 +283,37 @@ export default {
         </div>
       </div>
 
+      <h3 class="mt-20 mb-5 pb-5">
+        {{ t('branding.favicon.label') }}
+      </h3>
+      <label class="text-label">
+        {{ t('branding.favicon.tip', {}, true) }}
+      </label>
+
+      <div class="row mt-10 mb-20">
+        <Checkbox v-model="customizeFavicon" :label="t('branding.favicon.useCustom')" :mode="mode" />
+      </div>
+
+      <div v-if="customizeFavicon" class="row mb-20">
+        <div class="col logo-container span-12">
+          <div class="mb-10">
+            <FileSelector
+              :byte-limit="20000"
+              :read-as-data-url="true"
+              class="role-secondary"
+              :label="t('branding.favicon.upload')"
+              :mode="mode"
+              @error="setError"
+              @selected="updateLogo($event, 'uiFavicon')"
+            />
+          </div>
+          <SimpleBox v-if="uiFavicon" class="theme-light">
```

Comment:
For fear of introducing complexity, dark themes seem to come from the OS nowadays, so the tabs that favicons are in can be dark too. I don't think we need to have separate light/dark favicons, but maybe we should stick to the default background instead of forceing theme-light?

![image](https://user-images.githubusercontent.com/18697775/173086360-04fe68c1-dcc0-4d4f-837a-99e96b4473a1.png)